Makefile

```diff
@@ -115,6 +115,9 @@ bootstrap_zip: addon
 unsigned_bootstrap_xpi: bootstrap_zip
 	cp build/screenshots-bootstrap.zip build/screenshots.xpi
 
+dev_signed_bootstrap_xpi: bootstrap_zip
+	curl -v -F "input=@build/screenshots-bootstrap.zip" -o build/screenshots.xpi -H "Authorization: ${AUTOGRAPH_EDGE_TOKEN}" https://autograph-edge.stage.mozaws.net/sign
```

Comment:
Will `-v` result in the token being logged? Maybe we could pipe through sed or `grep -v` to filter it out and retain verbose output.

Reply:
Good catch. Let's see how far we can get without the `-v`